bool auto_pilot_in_control

